ci/scripts/utils/launch_java_agent.sh

```diff
@@ -56,22 +126,61 @@ cd "${JENKINS_AGENT_LANUCH_DIR}"
 
 if ! [[ -f agent.jar ]]; then
   curl -sO "${controller_url}/jnlpJars/agent.jar"
+  echo "Updated agent.jar downloaded"
+fi
```

Comment:
This will download `agent.jar` if it is not present, but what about updates? Should there be a CLI switch to force a download of the jar?

Reply:
No